I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script files/3_pursuit.rpy", line 2, in script
    stop black fadeout 1.0
  File "renpy/common/000statements.rpy", line 264, in execute_stop_music
    renpy.music.stop(fadeout=eval(p["fadeout"]), channel=channel)
Exception: Audio channel 'black' is unknown.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script files/3_pursuit.rpy", line 2, in script
    stop black fadeout 1.0
  File "C:\Users\mcboots42\Downloads\renpy-8.2.0-sdk\renpy\ast.py", line 1968, in execute
    self.call("execute")
  File "C:\Users\mcboots42\Downloads\renpy-8.2.0-sdk\renpy\ast.py", line 1950, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\mcboots42\Downloads\renpy-8.2.0-sdk\renpy\statements.py", line 349, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 264, in execute_stop_music
    renpy.music.stop(fadeout=eval(p["fadeout"]), channel=channel)
  File "C:\Users\mcboots42\Downloads\renpy-8.2.0-sdk\renpy\audio\music.py", line 298, in stop
    c = get_channel(channel)
  File "C:\Users\mcboots42\Downloads\renpy-8.2.0-sdk\renpy\audio\audio.py", line 928, in get_channel
    raise Exception("Audio channel %r is unknown." % name)
Exception: Audio channel 'black' is unknown.

Windows-10-10.0.22631 AMD64
Ren'Py 8.2.0.24012702
three peach hill 1.0
Sun Nov  3 20:02:48 2024
